main.go

```diff
@@ -149,20 +152,28 @@ func generateEnterpriseDownloadsJSON(semVerTag, appName string) enterpriseDownlo
 	d.Subscriptions["Enterprise"] = downloadsJSON{
 		Kubernetes: make(map[string]map[string]downloadJSON),
 		Linux:      make(map[string]map[string]downloadJSON),
-		MacOS:      make(map[string]map[string]downloadJSON),
+		Docker:     make(map[string]map[string]downloadJSON),
 		Windows:    make(map[string]map[string]downloadJSON),
+		MacOS:      make(map[string]map[string]downloadJSON),
 	}
 	for subscription := range d.Subscriptions {
-		d.Subscriptions[subscription].Linux["AIStor Object Store"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].MacOS["AIStor Object Store"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Windows["AIStor Object Store"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Linux["AIStor MinIO Client"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].MacOS["AIStor MinIO Client"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Windows["AIStor MinIO Client"] = map[string]downloadJSON{}
+		// Linux
+		d.Subscriptions[subscription].Linux["AIStor Server"] = map[string]downloadJSON{}
+		d.Subscriptions[subscription].Linux["AIStor Client"] = map[string]downloadJSON{}
 		d.Subscriptions[subscription].Linux["AIStor Key Manager"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Linux["AIStor Catalog"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Linux["AIStor Load Balancer"] = map[string]downloadJSON{}
-		d.Subscriptions[subscription].Kubernetes["AIStor"] = map[string]downloadJSON{}
+		d.Subscriptions[subscription].Linux["AIStor Loadbalancer/Firewall"] = map[string]downloadJSON{}
+		// Kubernetes
+		d.Subscriptions[subscription].Kubernetes["AIStor Server"] = map[string]downloadJSON{}
```

Comment:
Should we also add openshift here?

Reply:
We can under K8s section if we can maintain proper heirarchy.